computed/computedSignal.test.ts

```diff
@@ -0,0 +1,362 @@
+import type { InstalledClock } from '@sinonjs/fake-timers'
+import type { ReadableAtom, StoreValue } from '../index.js'
+
+import { equal, ok } from 'uvu/assert'
+import FakeTimers from '@sinonjs/fake-timers'
+import { test } from 'uvu'
+
+import { STORE_UNMOUNT_DELAY, computed, onMount, atom } from '../index.js'
+
+let clock: InstalledClock
+
+test.before(() => {
+  clock = FakeTimers.install()
+})
+
+test.after(() => {
+  clock.uninstall()
+})
+
+test('converts stores values', () => {
+  let letter = atom<{ letter: string }>({ letter: 'a' })
+  let number = atom<{ number: number }>({ number: 0 })
+
+  let renders = 0
+  let combine = computed(get => {
+    renders += 1
+    return `${get(letter).letter} ${number().number}`
+  })
+  equal(renders, 0)
+
+  let value: StoreValue<typeof combine> = ''
+  let unbind = combine.subscribe(combineValue => {
+    value = combineValue
+  })
+  equal(value, 'a 0')
+  equal(renders, 1)
+
+  letter.set({ letter: 'b' })
+  equal(value, 'b 0')
+  equal(renders, 2)
+
+  number.set({ number: 1 })
+  equal(value, 'b 1')
+  equal(renders, 3)
+
+  unbind()
+  clock.runAll()
+  equal(value, 'b 1')
+  equal(renders, 3)
+})
+
+test('works with single store', () => {
+  let number = atom<number>(1)
+  let decimal = computed(() => {
+    return number() * 10
```

Comment:
Do we want to change the api here? Maybe just use number.get()?

Reply:
The solid-js signal/memo api uses pure functions. It is a cleaner syntax but it does add some size to the bundle. I'm personally fine with either. I have an extension library (@ctx-core/nanostores) which adds a Proxy function over atoms & computeds.

Reply:
Oops, I didn't include the function api in the original issue. I'm ok removing the function api or leaving it in depending on what others think would be best.

Reply:
I removed the atom function that aliases .get in https://github.com/btakita/nanostores/pull/new/issues/188-no-fn

The tests which include example usages can be compared at:

https://github.com/btakita/nanostores/blob/issues/188-no-fn/computed/computedSignal.test.ts
https://github.com/btakita/nanostores/blob/issues/188/computed/computedSignal.test.ts